lib/openssl/misc.c

```diff
@@ -214,6 +214,8 @@ copy_val(const json_t *from, json_t *into, ...)
 static void __attribute__((constructor))
 constructor(void)
 {
+#if OPENSSL_API_COMPAT < 0x10100000L
```

Comment:
Sorry, I gave you the wrong instruction. You have to check for OpenSSL version number:
```suggestion
#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Reply:
Good point, thanks. I have updated it.